repositories.yaml

```diff
@@ -123,3 +123,19 @@ firefox-reality:
   dependencies:
     - org.mozilla.components:service-glean
     - org.mozilla.components:support-sync-telemetry
+lockwise-ios:
+  app_id: org.mozilla.ios.Lockbox
+  notification_emails:
+    - frank@mozilla.com
+    - tlong@mozilla.com
```

Comment:
I think this should contain also people from the Lockwise team, shouldn't it?

Reply:
Maybe `lockwise-dev@mozilla.com`?

Reply:
@travis79 can you confirm who the recipients should be?

Reply:
lockwise-dev@mozilla.com is good and maybe ddurst@mozilla.com?

Reply:
Done and done